frontend/app/src/py-handler.ts

```diff
@@ -288,6 +288,16 @@ export default class PyHandler {
 
   private terminateBackend = (client: ChildProcess) =>
     new Promise<void>((resolve, reject) => {
+      if (!client.pid) {
+        this.logToFile(
+          'subprocesses was already terminated (no process id pid found)'
```

Comment:
checking only one subprocess right?

```suggestion
          'subprocess was already terminated (no process id pid found)'
```